package.json

```diff
@@ -33,34 +33,37 @@
     "@babel/core": "^7.20.0",
     "@dev-plugins/apollo-client": "^0.0.6",
     "@expo-google-fonts/inter": "^0.2.3",
-    "@expo/vector-icons": "^14.0.3",
+    "@expo/vector-icons": "^14.0.2",
     "@react-native-async-storage/async-storage": "1.23.1",
-    "@react-navigation/native": "^6.0.2",
+    "@react-native-picker/picker": "2.9.0",
+    "@react-navigation/native": "^7.0.0",
     "@types/react-native-calendars": "^1.1264.7",
     "apollo-server": "^3.13.0",
     "axios": "^1.7.7",
     "date-fns": "^4.1.0",
-    "expo": "~51.0.37",
```

Comment:
why did you change expo version to 52 , i think we should use single expo version

Reply:
51 is not working i already said that.

Reply:
@Mbabazi12  We'll find a way it work on your side without changing main stream

Reply:
@MugemaneBertin2001 once you fix dev i will rebase with so for now i had to carry on my task with possible solution.